### DISCUSSION TITLE
Power 100 mW for use with another radio that performs the repeater function (My quansheng battery would like this)

### DISCUSSION THREAD

**Discussion by hssbo** [egzumer/uv-k5-firmware-custom#119] [Q&A]
Firstly, thank you for your dedication to building this firmware. It's the best!
It's just a suggestion: Many radio amateurs use a radio with the _CROSS BAND REPEATER_ function (VHF and UHF) and we transmit with the Quansheng from anywhere in our home or nearby. To have a battery that lasts longer, it would be interesting if a **100 mW power option** (only VHF and UHF) could be made available, as we are very close to the other radio that will do the retransmission. That would be really helpful for us! Thanks!

**Comment by ErikS-web** [comment]
The radio is optimized for use at 2 meters and 70 cm.
On other bands there is considerably less transmission power.

Use setting **L** for the lowest TX power. 
Can be changed via Long Press [`6 H/M/L`](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Button-functions#6-hml)

Tip: use [Chirp ](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Radio-operation#basic-operation--configuration)to set **POWER = Low** as default for your memory channels.

**Reply by hssbo** [reply]
You didn't understand my suggestion...

**Comment by ErikS-web** [comment]
READ the `README.md` under [user-customization.](https://github.com/egzumer/uv-k5-firmware-custom/tree/a43a3f949f32ed8ffbf0438face0edb7d093811e#user-customization)... there's the answer.

**ENABLE_REDUCE_LOW_MID_TX_POWER:= 0**       _makes medium and low power settings even lower_
It is already designed, but not active (set 0) in the Release(s).

So if you want to use it. Customize and Build your firmware with the instructions

**Comment by ErikS-web** [comment]
Tip. MEnu option 36. `BackLt` - backlight duration (**OFF**/ON/configured Time)

**Set it on OFF** and backlight of screen and keys stays off.
Even when a signal is received, you go through the menu or start broadcasting.
That will save power.

**Comment by egzumer** [comment]
As Erik mentioned there is a reduced M/L power levels build option. Not 100mW but it will lower power levels considerably. Other than that, if I'm going to modify power levels this would be added as custom power levels under current menu, but it wouldn't be compatible with chirp nor with CPS, so it is just under consideration for now.

**Comment by ErikS-web** [comment]
What can be tuned? Because **_the output power is not constant at all frequencies_**. This picture (recently on Telegram) makes that clear.

![Power over the frequency 136 - 468 Mhz - photo_2023-05-26_09-17-42](https://github.com/egzumer/uv-k5-firmware-custom/assets/148579604/ab4e7621-94a9-4f5b-967a-5c2bfbb96a85)

I can't check the picture myself, because I don't have a spectrum meter, only a power meter (up to 150 Mhz)

**Reply by egzumer** [reply]
The output power can have any value between 0-63 (-24.3-7.26 dBm), but I'm not going to mess with that. Power output across the spectrum is calibrated at the factory. What I would do, if anything, would be dividing calibrated values by a some constant value.

**Comment by ErikS-web** [comment]
For me, the battery lasts all day and I put it on the charger overnight.

But if you want to use it for 2 days... you can purchase a **_UV-5R battery of 3800mAh_** (Baofeng BL-5) and fit it in the UV-K5. Requires clipping 1 edge.
